packages/components/src/lib/distributionSpecBuilder.ts

```diff
@@ -1,22 +1,25 @@
 import { VisualizationSpec } from "react-vega";
+import * as yup from "yup";
 import type { LogScale, LinearScale, PowScale, TimeScale } from "vega";
 
-export type DistributionChartSpecOptions = {
-  /** Set the x scale to be logarithmic by deault */
-  logX: boolean;
-  /** Set the y scale to be exponential by deault */
-  expY: boolean;
-  /** The minimum x coordinate shown on the chart */
-  minX?: number;
-  /** The maximum x coordinate shown on the chart */
-  maxX?: number;
-  /** The title of the chart */
-  title?: string;
-  /** The formatting of the ticks */
-  format?: string;
-  /** Whether the x-axis should be dates or numbers */
-  xAxisType?: "number" | "dateTime";
-};
```

Comment:
Here and in a few other places you removed the comments together with props.

The comments seem pretty useful, they should be moved to yup schema definitions (as comments, I there's no `.description()` in yup) instead.